### ISSUE TITLE
Cannot set the contentDescription of the dialog buttons programmatically to allow accessibility

### ISSUE THREAD

**Issue by pivotal-steve-chepurny** [afollestad/material-dialogs#1386]
- [X ] I have verified there are no duplicate active or recent bugs, questions, or requests
- [X ] I have verified that I am using the latest version of Material Dialogs.
- [X ] I have given my issue a non-generic title.
- [X ] I have read over the documentation (before asking questions on how to do something).

###### Include the following:
 - Material Dialogs version: `0.9.4.5`
 - Device OS version: `7.1.2`
 - Device Manufacturer: `Google`
 - Device Name: `Pixel`

Also, please wrap Java with correct syntax highlighting.

Can no longer set the contentDescription for the buttons when we extend MaterialDialog as they are now private. Example that we were using before:

```java
 if (positiveButton != null) {
            positiveButton.setContentDescription(positiveButton.getText() + " " + getContext().getString(R.string.accessibility_button));
        }
```

**Comment by afollestad** [comment]
https://github.com/afollestad/material-dialogs#getting-and-setting-action-buttons

**Comment by AndreasDolphin** [comment]
As the old readme has been deprecated, here is an updated link: https://github.com/afollestad/material-dialogs/blob/master/README_OLD.md#getting-and-setting-action-buttons
